src/main/java/uk/gov/hmcts/reform/sscs/ccd/domain/FtaCommunicationFields.java

```diff
@@ -27,6 +27,10 @@ public class FtaCommunicationFields {
     private String ftaRequestNoResponseTextArea;
     private List<String> ftaRequestNoResponseNoAction;
 
+    private DynamicList ftaResponseNoActionedRadioDl;
+    private String ftaResponseNoActionedQuery;
+    private  YesNo ftaResponseActioned;
```

Comment:
add a reply field here too to instantiate and use in the event along with the query, 
see https://github.com/hmcts/sscs-common/pull/1739/files#diff-913f3343e9f2cbfceba37e2b018b2c7f598e922cd83eb5a52f14767b80f160b8
and https://github.com/hmcts/sscs-tribunals-case-api/pull/4644/files#diff-3b73fc7a8bbf36704f3a830b5d9625c87cc0ef744a79d91eb5b9d7685c480165

Reply:
Done